main/domoticz.cpp

```diff
@@ -884,6 +886,21 @@ int main(int argc, char**argv)
 	{
 		g_bDontCacheWWW = true;
 	}
+	if (cmdLine.HasSwitch("-wwwcompress"))
+	{
+		if (cmdLine.GetArgumentCount("-wwwcompress") != 1)
+		{
+			_log.Log(LOG_ERROR, "Please specify a compress mode");
+			return 1;
+		}
+		std::string szmode = cmdLine.GetSafeArgument("-wwwcompress", 0, "on");
+		if (szmode == "off")
+			g_wwwCompressMode = 0xFF;
+		else if (szmode == "static")
+			g_wwwCompressMode = 0x0;
+		// should I verify possible invalid input? doesn't seem to be done with other parameters
```

Comment:
Please do not add comments/questions in a PR
Always good to discuss it
No you do not as you set the default at the top of the code

Reply:
So invalid equals default? As the comment states: parameter validation doesn't seem to be done on other inputs as well. Was more of a remark to myself and didn't really think about it when entering the PR (which in this case is mostly cut-and-paste from the original PR)